src/components/grid.js

```diff
@@ -1,114 +1,125 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
I notice that we've added prop-types for two of the components, which is great, but not for `gridContainer.js`. Is there a reason for this? If not, it'd be good to be consistent and use prop-types on all React components.

Reply:
yes this will be added ...and  GridContainer is being reworked  as  preview is being moved into its own component so should be there in net feature PR

Reply:
Sounds sensible 👍